### DISCUSSION TITLE
issue running the rocket.chat in locally

### DISCUSSION THREAD

**Discussion by rajaNayak123** [RocketChat/Rocket.Chat#34093] [Q&A]
The issue:
When running the setup, I get stuck at the step where the process says: "watching for file changes".
 OS: macOS (M1 chip)
 Node.js version: v20.18.0
 
<img width="1440" alt="Screenshot 2024-11-30 at 7 14 49 PM" src="https://github.com/user-attachments/assets/c797d112-a024-435b-ad04-038a31870330">
<hr>
<img width="1440" alt="Screenshot 2024-11-30 at 10 05 57 PM" src="https://github.com/user-attachments/assets/40a4c791-da02-480c-bf36-ac42d175d8f4">

I’d be very grateful if you could guide me on how to resolve this

**Comment by OrneyVortex** [comment]
try clearing cache and then restart install all the dependencies again!
(worked for me once)

**Comment by reetp** [comment]
The correct places to ask these questions are here:

https://open.rocket.chat/channel/support/
https://forums.rocket.chat/
